spec/spec_helper.rb

```diff
@@ -1,51 +1,39 @@
+# frozen_string_literal: true
+
 require 'pathname'
 dir = Pathname.new(__FILE__).parent
 $LOAD_PATH.unshift(dir, File.join(dir, 'fixtures/modules/augeasproviders_core/spec/lib'), File.join(dir, '..', 'lib'))
 
-require 'rubygems'
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
-require 'simplecov'
-unless RUBY_VERSION =~ /^1\.8/
-  require 'coveralls'
-  SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-end
-SimpleCov.start do
-  add_group "Puppet Types", "/lib/puppet/type/"
-  add_group "Puppet Providers", "/lib/puppet/provider/"
+# puppetlabs_spec_helper will set up coverage if the env variable is set.
+# We want to do this if lib exists and it hasn't been explicitly set.
+ENV['COVERAGE'] ||= 'yes' if Dir.exist?(File.expand_path('../lib', __dir__))
 
-  add_filter "/spec/fixtures/"
-  add_filter "/spec/unit/"
-  add_filter "/spec/support/"
```

Comment:
Is this now properly fixed in puppetlabs_spec_helper? This is one of the few modules where coverage may actually be useful.

Reply:
If I remember it correctly, puppetlabs_spec_helper does this properly now.

Reply:
https://github.com/puppetlabs/puppetlabs_spec_helper/blob/main/spec/spec_helper.rb#L3-L24 yep